src/unity/python/turicreate/data_structures/image.py

```diff
@@ -184,6 +184,19 @@ def __repr__(self):
         ret = ret + "Channels: " + str(self._channels) + "\n"
         return ret
 
+    def _repr_png_(self):
+        img = self._to_pil_image()
+        from io import BytesIO
+        b = BytesIO()
+        img.save(b, format='png')
+        data = b.getvalue()
+        res = {"Height" :str(self._height), "Width":str(self._width), "Channels: " :str(self._channels)}
+        return (data,res)
+
+        # ip_img = display.Image(data=data, format='png', embed=True)
```

Comment:
I wouldn't check in commented out code without a good reason. If there is a good reason to have it, please add a comment about why it's important.

Reply:
Got it.